2021-10-14 19:12:50,666 - 8768 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: No module named 'websockets.legacy'
Traceback (most recent call last):
  File "C:\Users\geral\hummingbot\hummingbot\core\utils\async_utils.py", line 9, in safe_wrapper
    return await c
  File "C:\Users\geral\hummingbot\hummingbot\core\utils\trading_pair_fetcher.py", line 47, in fetch_all
    module = getattr(importlib.import_module(module_path), class_name)
  File "C:\Users\geral\anaconda3\envs\hummingbot\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\geral\hummingbot\hummingbot\connector\exchange\kucoin\kucoin_api_order_book_data_source.py", line 24, in <module>
    from websockets.legacy.client import Connect as WSConnectionContext
ModuleNotFoundError: No module named 'websockets.legacy'
2021-10-14 19:20:19,026 - 7648 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: No module named 'websockets.legacy'
Traceback (most recent call last):
  File "C:\Users\geral\hummingbot\hummingbot\core\utils\async_utils.py", line 9, in safe_wrapper
    return await c
  File "C:\Users\geral\hummingbot\hummingbot\core\utils\trading_pair_fetcher.py", line 47, in fetch_all
    module = getattr(importlib.import_module(module_path), class_name)
  File "C:\Users\geral\anaconda3\envs\hummingbot\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\geral\hummingbot\hummingbot\connector\exchange\kucoin\kucoin_api_order_book_data_source.py", line 24, in <module>
    from websockets.legacy.client import Connect as WSConnectionContext
ModuleNotFoundError: No module named 'websockets.legacy'
